tests/docker/test-docker-compose.yml

```diff
@@ -46,12 +46,10 @@ services:
       - ./.env
     depends_on:
       - bridge0
-    # FIXME - use a tagged version eventually
-#    image: "ghcr.io/astriaorg/go-ethereum:latest"
-#    image: "ghcr.io/astriaorg/go-ethereum:local"
-    image: "ghcr.io/astriaorg/go-ethereum:custom-mempool"
+    image: "ghcr.io/astriaorg/go-ethereum:0.0.3"
```

Comment:
does this need to be updated for this PR? https://github.com/astriaorg/go-ethereum/pull/13 or does it already use it

Reply:
I've pushed the tag before the PR gets merged. I don't love doing that though. I need to think a bit more on how semver bumps can be done more painlessly. I also had to push the same tag for metro to keep same versions compatible.